api/src/main/kotlin/edu/wgu/osmt/collection/CollectionController.kt

```diff
@@ -36,210 +23,298 @@ import org.springframework.security.core.annotation.AuthenticationPrincipal
 import org.springframework.security.oauth2.jwt.Jwt
 import org.springframework.stereotype.Controller
 import org.springframework.transaction.annotation.Transactional
-import org.springframework.web.bind.annotation.DeleteMapping
-import org.springframework.web.bind.annotation.GetMapping
-import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.PostMapping
-import org.springframework.web.bind.annotation.RequestBody
-import org.springframework.web.bind.annotation.RequestMapping
-import org.springframework.web.bind.annotation.RequestParam
-import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.*
 import org.springframework.web.server.ResponseStatusException
 import org.springframework.web.util.UriComponentsBuilder
 
 @Controller
 @Transactional
 class CollectionController @Autowired constructor(
-    val collectionRepository: CollectionRepository,
-    val richSkillRepository: RichSkillRepository,
-    val taskMessageService: TaskMessageService,
-    val auditLogRepository: AuditLogRepository,
-    val collectionEsRepo: CollectionEsRepo,
-    val appConfig: AppConfig,
-    val oAuthHelper: OAuthHelper
-): HasAllPaginated<CollectionDoc> {
+        val collectionRepository: CollectionRepository,
+        val richSkillRepository: RichSkillRepository,
+        val taskMessageService: TaskMessageService,
+        val auditLogRepository: AuditLogRepository,
+        val collectionEsRepo: CollectionEsRepo,
+        val appConfig: AppConfig,
+        val oAuthHelper: OAuthHelper
+) : HasAllPaginated<CollectionDoc> {
     override val elasticRepository = collectionEsRepo
-
-    override val allPaginatedPath: String = RoutePaths.COLLECTIONS_LIST
+    override val allPaginatedPath: String = "${RoutePaths.LATEST}${RoutePaths.COLLECTIONS_LIST}"
     override val sortOrderCompanion = CollectionSortEnum.Companion
-
-    @GetMapping(RoutePaths.COLLECTIONS_LIST, produces = [MediaType.APPLICATION_JSON_VALUE])
+    
+    @GetMapping(path = [
+        "${RoutePaths.VERSIONED_API}${RoutePaths.COLLECTIONS_LIST}",
+        "${RoutePaths.UNVERSIONED_API}${RoutePaths.COLLECTIONS_LIST}"
```

Comment:
Is accepting any version e.g (v4, v5, string-version)

Probably should be `"${RoutePaths.API}${RoutePaths.LATEST}${RoutePaths.COLLECTIONS_LIST}" `

Reply:
yeah, @JohnKallies, @Corpratespaz, thoughts on this?